tests/queries/0_stateless/01710_minmax_count_projection_distributed_query.sql

```diff
@@ -0,0 +1,7 @@
+-- Tags: no-s3-storage
```

Comment:
But there is no projection, so "no-s3-storage" tag and "projection" from the name can be removed.

Reply:
There is `minmax_count_projection` which is used. I'll remove "no-s3-storage" tag as this projection works with s3 storage.